Modules/ITS/src/TrendingTaskITSFEE.cxx

```diff
@@ -0,0 +1,356 @@
+  mTrend->Draw("time", "", "goff");
+  std::vector<double> retrieveTime(mTrend->GetV1(), mTrend->GetV1() + mTrend->GetSelectedRows() % mTrend->GetEstimate());
+
+  std::vector<double> retrieveRunNumber;
+  for (int j = 1; j <= numberOfEntries; j++)
+    retrieveRunNumber.push_back((double)j);
+
+  // Lane status summary plots
+  for (const auto& plot : mConfig.plots) {
+
+    if (plot.varexp.find("binContent") == std::string::npos)
+      continue;
+
+    // Checks neeeded to segregate plots from input list
+    if (countplots > nFlags - 1) {
+      countplots = 0;
+      countITSpart++;
+    }
+
+    // Retrieve data for trend plot
+    mTrend->Draw(plot.varexp.c_str(), "", "goff");
+    std::vector<double> dataRetrieve(mTrend->GetV1(), mTrend->GetV1() + mTrend->GetSelectedRows() % mTrend->GetEstimate());
+
+    // Initialize MultiGraph and Legend
+    if (countplots == 0) {
+      multi_trend = new TMultiGraph();
+      SetGraphName(multi_trend, plot.name, Form("Lane status summary trend %s", itsParts[countITSpart].c_str()));
+
+      legend = new TLegend(0.93, 0.55, 1.0, 0.9);
+      SetLegendStyle(legend, Form("LaneStatusSummary_%s_legend", itsParts[countITSpart].c_str()));
+
+      isRun = plot.selection.find("Run") != std::string::npos ? true : false;
+    }
+
+    trend_plot_Flags[countplots] = new TGraph(numberOfEntries, isRun ? &retrieveRunNumber[0] : &retrieveTime[0], &dataRetrieve[0]);
+    trend_plot_Flags[countplots]->SetName(plot.name.c_str());
+    SetGraphStyle(trend_plot_Flags[countplots], colors[countplots], markers[countplots]);
+    multi_trend->Add(trend_plot_Flags[countplots], plot.option.c_str());
+
+    legend->AddEntry(trend_plot_Flags[countplots], Form("%s", trend_titles[countplots].c_str()), "pl");
+
+    // Create and save plots
+    if (countplots == nFlags - 1) {
+
+      SetGraphAxes(multi_trend, Form("%s", isRun ? "Run" : "Time"), "Number of lanes", !isRun);
+
+      // Canvas settings
+      std::string name = "LaneStatusSummary_" + itsParts[countITSpart] + "_Trends";
+      canvas = new TCanvas(Form("%s", name.c_str()), Form("%s", name.c_str()));
+      SetCanvasSettings(canvas);
+
+      // Plot as a function of run number requires a dummy TH1 histogram
+      TH1I* hDummy = nullptr;
+      if (isRun) {
+        hDummy = new TH1I("hDummy", Form("%s; %s; %s", multi_trend->GetTitle(), multi_trend->GetXaxis()->GetTitle(), multi_trend->GetYaxis()->GetTitle()), numberOfEntries, 0.5, numberOfEntries + 0.5);
+        SetHistoAxes(hDummy, runlist, multi_trend->GetYaxis()->GetXmin(), multi_trend->GetYaxis()->GetXmax());
+      }
+
+      // Draw plots
+      if (hDummy)
+        hDummy->Draw();
+      multi_trend->Draw(Form("%s", hDummy ? "" : "a"));
+      legend->Draw("SAME");
+
+      // Upload plots
+      auto mo = std::make_shared<MonitorObject>(canvas, mConfig.taskName, "o2::quality_control_modules::its::TrendingTaskITSFEE", mConfig.detectorName);
+      mo->setIsOwner(false);
+      qcdb.storeMO(mo);
```

Comment:
If I understand correctly, here you store to ccdb every single plot as defined in the json. If so, can you remove this and keep only the `storeMO` related to the storage of the canvases with all trends together?